test/test_binary_ufuncs.py

```diff
@@ -3026,6 +3026,14 @@ def test_muldiv_scalar(self, device, dtype):
         self.assertEqual(x / s, x / y)
         self.assertEqual(s / x, y / x)
 
+        if dtype == torch.half:
```

Comment:
there's `test_divmul_scalar` that runs only on cuda currently that you should be able to make run on cpu, and not change this test?

Reply:
Oh, thanks for the remainder, did not notice this one before.